docs/examples/trees_example.py

```diff
@@ -0,0 +1,232 @@
+import trees
+from fury import window, actor, lib
+import numpy as np
+
+
```

Comment:
Please, check the other examples. You need to include some documentation for the example. For instance, explaining in details what is happening in the code, why you are doing it and how to do it.